osf/utils/workflows.py

```diff
@@ -355,6 +378,45 @@ def db_name(self):
     },
 ]
 
+
+COLLECTION_SUBMISSION_TRANSITIONS = [
+    {
```

Comment:
will need an `IN_PROGRESS -> ACCEPTED` transition dependent on whether the provider is moderated